Prototyper/Classes/Feedback/EditScreenshotView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  EditScreenshotView.swift
+//  Prototyper
+//
+//  Created by Raymond Pinto on 03.01.20.
+//
+
+import SwiftUI
+
+struct Drawing {
+    var points: [CGPoint] = []
+}
+
+struct EditScreenshotView: View {
+    @EnvironmentObject var model: Model
+    @Environment(\.presentationMode) var presentationMode
+    @Environment(\.colorScheme) var colorScheme
+    @State var currentDrawing: Drawing = Drawing()
+    @State var drawings: [Drawing] = [Drawing]()
+    @State var rect: CGRect = .zero
+    @State var color: Color = Color.black
+    @State var colorPickerShown: Bool = false
+    
+    var body: some View {
+        VStack(alignment: .center) {
+            GeometryReader { geometry in
```

Comment:
I would suggest to add some spacer in before the GeometryReader:
```suggestion
            GeometryReader { geometry in
                Spacer(minLength: 16)
```